Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -76,6 +90,11 @@ public void stop() {
         playersReady.clear();
     }
 
+    /**
+     * Sends the WorldObjects to the player.
+     *
+     * @param player The given player, not null.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)